tests/test_script.py

```diff
@@ -436,3 +436,47 @@ def test_ipython():
     )
     res = subprocess.check_output(["lineapy", "ipython", "-c", raw_code])
     assert res.decode().strip().endswith(clean_code)
+
+
+@pytest.mark.slow
+def test_lineapy_upgrade_db():
+    """
+    Verify that `lineapy upgrade-db` brings db up to date.
```

Comment:
I'm not following how this tests that the upgrade brings the db up to date but rather that the subprocess doesn't crash? I'm fine with the latter, just making sure I understand.

Reply:
that's right. this only checks if the command crashes

Reply:
I've updated the comment to reflect this.